docs/freefeatures.rst

```diff
@@ -222,12 +222,11 @@ of using ``pyscript`` is shown below  along with the **examples/arg_printer.py**
     If you want to be able to pass arguments with spaces to scripts, then we strongly recommend using one of the decorators,
```

Comment:
This note seems out of place. It says: "If you want to be able to pass arguments with spaces to scripts..."
But `lprint` is not a script.
Maybe just delete this note since the `with_argument_list` decorator is already documented elsewhere.

Reply:
Thanks for pointing that out.  I will reword it.